### PR TITLE
Grammar Change

### PR DESCRIPTION
**Pull request by PhilipElder**
It's " ... needs a client access license (CAL)."
~~"a Remote Desktop Server" <-- Acronyms are treated based on the first _word_ not letter.~~
^^^ My apologies, I stand corrected as per our conversation. A and An are based on pronunciation not what the acronym stands for. :)
A couple more tweaks.
Added the Per Device and Per User examples for RDS CALs.

### CODE REVIEW

File: WindowsServerDocs/remote/remote-desktop-services/rds-client-access-license.md [hunk 1]
```diff
@@ -52,6 +52,8 @@ When you use the Per Device model, a temporary license is issued the first time
 
 When you use the Per User model, licensing is not enforced and each user is granted a license to connect to an RD Session Host from any number of devices. The license server issues licenses from the available RDS CAL pool or the Over-Used RDS CAL pool. It's your responsibility to ensure that all of your users have a valid license and zero Over-Used CALs—otherwise, you're in violation of the Remote Desktop Services license terms.
 
+An example of where one would use the Per Device model would be in an environment where there are two more more shifts using the same computers to access the RD Session Host(s). The Per User model would be best for environments where users have their own dedicated Windows device to access the RD Session Host(s).
```
Comment (illfated) [inline WindowsServerDocs/remote/remote-desktop-services/rds-client-access-license.md]: "more more" ;-)
```suggestion
An example of where one would use the Per Device model would be in an environment where there are two more shifts using the same computers to access the RD Session Host(s). The Per User model would be best for environments where users have their own dedicated Windows device to access the RD Session Host(s).
```

**Comment by illfated** [comment]
> "a Remote Desktop Server" <-- Acronyms are treated based on the first _word_ not letter.

Interesting point, I was not fully aware of this. Thank you for the useful tip.

File: WindowsServerDocs/remote/remote-desktop-services/rds-client-access-license.md (hunk 1 above)
Comment (PhilipElder) [reply WindowsServerDocs/remote/remote-desktop-services/rds-client-access-license.md]: Oops ... this:

An example of where one would use the Per Device model would be in an environment where there are two **or** more shifts using the same computers to access the RD Session Host(s).

Comment (illfated) [reply WindowsServerDocs/remote/remote-desktop-services/rds-client-access-license.md]: Oh. Right. I will edit my suggestion, just so you can commit my suggestion with the corrected content.

Comment (illfated) [reply WindowsServerDocs/remote/remote-desktop-services/rds-client-access-license.md]: Edited. Proceed at your leisure.

**Comment by illfated** [comment]
@JohanFreelancer9 : I am not quite sure if this repository is part of your work area, so would it be OK if I ask you for a copy review here?

**Comment by PhilipElder** [comment]
I'm not sure what the question means?

**Comment by illfated** [comment]
Oh. Right. We usually check with the regular copy reviewers (@JohanFreelancer9 & @nenonix, among others) to ask for an extra verification that the document is cleared for merging (nothing more to improve upon). Maybe I am being overzealous, so we may just move on without the extra copy review, seeing that Windows-ITPro-Docs is a different repository with different reviewers, authors and managers.

**Comment by illfated** [comment]
#sign-off

**Comment by eross-msft** [comment]
@Heidilohr  Please review these changes and let me know if it's ready to merge.

File: WindowsServerDocs/remote/remote-desktop-services/rds-client-access-license.md [hunk 2]
```diff
@@ -18,11 +18,11 @@ manager: dongill
...
+Each user and device that connects to a Remote Desktop Session host needs a client access license (CAL). You use RD Licensing to install, issue, and track RDS CALs.  
 
-When a user or a device connects to an RD Session Host server, the RD Session Host server determines if an RDS CAL is needed. The RD Session Host server then requests an RDS CAL from the Remote Desktop license server. If an appropriate RDS CAL is available from a license server, the RDS CAL is issued to the client, and the client is able to connect to the RD Session Host server and from there to the desktop or apps they're trying to use.
+When a user or a device connects to an RD Session Host server, the RD Session Host server determines if a RDS CAL is needed. The RD Session Host server then requests a RDS CAL from the Remote Desktop license server. If an appropriate RDS CAL is available from a license server, the RDS CAL is issued to the client, and the client is able to connect to the RD Session Host server and from there to the desktop or apps they're trying to use.
```
Comment (JohanFreelancer9) [inline WindowsServerDocs/remote/remote-desktop-services/rds-client-access-license.md]:
```suggestion
When a user or a device connects to an RD Session Host server, the RD Session Host server determines if an RDS CAL is needed. The RD Session Host server then requests an RDS CAL from the Remote Desktop license server. If an appropriate RDS CAL is available from a license server, the RDS CAL is issued to the client, and the client is able to connect to the RD Session Host server and from there to the desktop or apps they're trying to use.
```

File: WindowsServerDocs/remote/remote-desktop-services/rds-client-access-license.md [hunk 3]
```diff
@@ -52,6 +52,8 @@ When you use the Per Device model, a temporary license is issued the first time
 
 When you use the Per User model, licensing is not enforced and each user is granted a license to connect to an RD Session Host from any number of devices. The license server issues licenses from the available RDS CAL pool or the Over-Used RDS CAL pool. It's your responsibility to ensure that all of your users have a valid license and zero Over-Used CALs—otherwise, you're in violation of the Remote Desktop Services license terms.
 
+An example of where one would use the Per Device model would be in an environment where there are two or more shifts using the same computers to access the RD Session Host(s). The Per User model would be best for environments where users have their own dedicated Windows device to access the RD Session Host(s).
+
 To ensure you are in compliance with the Remote Desktop Services license terms, track the number of RDS Per User CALs used in your organization and be sure to have a enough RDS Per User CALs installed on the license server for all of your users.
```
Comment (JohanFreelancer9) [inline WindowsServerDocs/remote/remote-desktop-services/rds-client-access-license.md]:
```suggestion
To ensure you are in compliance with the Remote Desktop Services license terms, track the number of RDS Per User CALs used in your organization and be sure to have enough RDS Per User CALs installed on the license server for all of your users.
```

File: WindowsServerDocs/remote/remote-desktop-services/rds-client-access-license.md (hunk 2 above)
Comment (illfated) [reply WindowsServerDocs/remote/remote-desktop-services/rds-client-access-license.md]: From the PR author:
> "a Remote Desktop Server" <-- Acronyms are treated based on the first word not letter.

(see also my comment https://github.com/MicrosoftDocs/windowsserverdocs/pull/3946#issuecomment-590071034 and the main PR comment  https://github.com/MicrosoftDocs/windowsserverdocs/pull/3946#issue-378668444 )

Comment (illfated) [reply WindowsServerDocs/remote/remote-desktop-services/rds-client-access-license.md]: (@PhilipElder : Feel free to mark this conversation as Resolved)

Comment (illfated) [reply WindowsServerDocs/remote/remote-desktop-services/rds-client-access-license.md]: Mind you, from my own experience, English speech usually adapts to how acronyms are pronounced.
https://blog.apastyle.org/apastyle/2012/04/using-a-or-an-with-acronyms-and-abbreviations.html
https://www.grammar.com/a-vs-an-when-to-use/

Comment (PhilipElder) [reply WindowsServerDocs/remote/remote-desktop-services/rds-client-access-license.md]: Neat, so I sit corrected as I dig around:
https://ontariotraining.net/grammar-tip-a-or-an-with-acronyms/
https://www.grammar.com/a-vs-an-when-to-use/

Things change. :)

Comment (illfated) [reply WindowsServerDocs/remote/remote-desktop-services/rds-client-access-license.md]: 😄 Fair enough, glad it was a light-hearted debate (tempered debates tend to make me feel less helpful & useful).
I presume from your reply that we can treat "RDS" as an _initialism_ in this case. :)
Looking forward to your merge (commit) of that change. BTW, you may want to edit your PR comment too.

Comment (PhilipElder) [reply WindowsServerDocs/remote/remote-desktop-services/rds-client-access-license.md]: I'm just learning GitHub and its language/semantics.

Which "PR" comment do I need to edit? The one with the a/an RDS?

Comment (illfated) [reply WindowsServerDocs/remote/remote-desktop-services/rds-client-access-license.md]: Oh. Right. I should have been more precise. Your initial PR opening statement:

https://github.com/MicrosoftDocs/windowsserverdocs/pull/3946 (top of page) | https://github.com/MicrosoftDocs/windowsserverdocs/pull/3946#issue-378668444 (1st comment)
> It's " ... needs a client access license (CAL)."
> "a Remote Desktop Server" <-- Acronyms are treated based on the first _word_ not letter.
> A couple more tweaks.
> Added the Per Device and Per User examples for RDS CALs.

**Comment by JohanFreelancer9** [comment]
For reference (from the Microsoft Style Guide) https://docs.microsoft.com/en-us/style-guide/acronyms#use-a-or-an-depending-on-pronunciation

**Comment by illfated** [comment]
Interesting, thanks for the link. I suggest we go for consistency (regarding RD and RDS).

Just as an unrelated comment, I also noticed that SQL uses "an SQL database" as in pronouncing each letter individually, "ess-cue-ell" (which I have always done), but I also know that quite a few programmers have learnt to pronounce it "sequel" based on some old-school reference (which I can't quite remember at the moment).

**Comment by JohanFreelancer9** [comment]
By the way, some interesting reading on the pronunciation, use, and origins of SQL and Sequel.

**Comment by PhilipElder** [comment]
> 
> 
> For reference (from the Microsoft Style Guide) https://docs.microsoft.com/en-us/style-guide/acronyms#use-a-or-an-depending-on-pronunciation. In the same paragraph we now have "an RD Session Host server" but "a RDS CAL".

Thanks for this.

**Comment by PhilipElder** [comment]
> 
> 
> By the way, some interesting reading on the pronunciation, use, and origins of SQL and Sequel.

This one can almost get "religious" depending on who I talk to. ;)

I've taken to Sequel (Pronounced: SeaQwill) since that's what I started with oh so many years ago when I got into IT.

**Comment by PhilipElder** [comment]
Thank you for the great conversation and the guidance around the proper grammar for acronyms.

**Comment by PhilipElder** [comment]
Did I commit the changes correctly? N00b here sorry.

File: WindowsServerDocs/remote/remote-desktop-services/rds-client-access-license.md (hunk 2 above)
Comment (illfated) [inline WindowsServerDocs/remote/remote-desktop-services/rds-client-access-license.md]: Looks like you marked the previous conversation here as "Resolved" while not committing the changes. Let me re-add the commit suggestion:
```suggestion
When a user or a device connects to an RD Session Host server, the RD Session Host server determines if an RDS CAL is needed. The RD Session Host server then requests an RDS CAL from the Remote Desktop license server. If an appropriate RDS CAL is available from a license server, the RDS CAL is issued to the client, and the client is able to connect to the RD Session Host server and from there to the desktop or apps they're trying to use.
```
(This is the part about pronouncing RDS as "ar-dee-ess" when reading it aloud.)

**Comment by illfated** [comment]
Very good. Looks like it is very close to being complete, at least as far as concerning the reviews from @JohanFreelancer9 and me.

**Comment by illfated** [comment]
(@JohanFreelancer9 : Feel free to add your approval once you have checked out the latest changes.)

**Comment by illfated** [comment]
#sign-off

**Pull request merged**